app/server/index.js

```diff
@@ -220,7 +220,7 @@ app.prepare().then(() => {
   server.use(
     postgraphile(pgPool, process.env.DATABASE_SCHEMA || 'ggircs_portal', {
       appendPlugins: [PgManyToManyPlugin],
-      graphiql: true,
+      graphiql: process.env.IS_LOCAL_ENVIRONMENT === 'true',
```

Comment:
```suggestion
      graphiql: process.env.NODE_ENV !== 'production',
```

Reply:
won't this allow graphiql to run in test.ciip? Which we don't want?

Reply:
nevermind, it won't. I'll make the change